pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -47,6 +47,7 @@
   private static final String DELETED_SEGMENTS_RETENTION_IN_DAYS = "controller.deleted.segments.retentionInDays";
   private static final String TASK_MANAGER_FREQUENCY_IN_SECONDS = "controller.task.frequencyInSeconds";
   private static final String TABLE_MIN_REPLICAS = "table.minReplicas";
+  private static final String ACCEPT_SPLIT_COMMIT = "controller.accept.split.commit";
```

Comment:
maybe better to call both of them as enable.split.commit (so, change the one in the controller to be similar to the config in the server). thanks